templates/partials/banner.html

```diff
@@ -1,5 +1,6 @@
 <script src="//unpkg.com/globe.gl"></script>
 <div class="row mb-3 banner-text-container">
+    {% include "partials/alert.html" %}
```

Comment:
I don't think that the banner partial is the right place for this. Not all pages have a banner (ie. the tutorials page). 

I think that it should go after the menu partial declaration on the layouts/main.html file:

```
<body>
{% include "partials/menu.html" %}
{% include "partials/alert.html" %}
{% block content %} {% endblock %}
{% include "partials/footer.html" %}
</body>
```

Reply:
Because of how the HTML is done putting it here will push down the banner instead of floating on top of it.

Reply:
Moved alert partial to main.html